### ISSUE TITLE
Cannot read property 'resourceStart' of undefined

### ISSUE THREAD

**Issue by andytuch** [nexe/nexe#578]
**Is this a BUG or a FEATURE REQUEST?:** 
Potential Bug? 

**What happened:**
When compiling my node project with nexe, I sign it and change the rc details. In order to do this successfully, I use the patches listed in the issue at https://github.com/nexe/nexe/issues/446. After successfully building the EXE, I get the error below when running it.

> _third_party_main.js:25
>     var _a = binary.layout, resourceStart = _a.resourceStart, stat = _a.stat;
>                                                ^
> 
> TypeError: Cannot read property 'resourceStart' of undefined
>     at shimFs (_third_party_main.js:25:48)
>     at _third_party_main.js:309:1
>     at _third_party_main.js:309:25
>     at NativeModule.compile (bootstrap_node.js:622:7)
>     at Function.NativeModule.require (bootstrap_node.js:567:18)
>     at bootstrap_node.js:142:22
>     at _combinedTickCallback (internal/process/next_tick.js:131:7)
>     at process._tickCallback (internal/process/next_tick.js:180:9)

**What you expected to happen:**
Expect the EXE to run my app properly

**How to reproduce it (as minimally and precisely as possible):**
```
nexe.compile({
    input: 'client.js',
    output: FILENAME,
    build: true,
    temp: './.node',
    ico: './res/icon.ico',
    verbose: false,
    target: 'win32-x86-8.11.4',
    rc: {
      "CompanyName": 'XXXXX',
      "ProductName": 'XXXXX',
      "FileDescription": 'XXXXX',
      "FileVersion": VERSION,
      "ProductVersion": VERSION,
      "FILEVERSION": VERSION,
      "PRODUCTVERSION": VERSION,
      "OriginalFilename": 'client.exe',
      "InternalName": 'client',
      "LegalCopyright": ''
    },
    patches: [
      (x, next) => {
        x.code = () => [x.shims.join(''), x.input].join(';')
        return next()
      },
     async (compiler, next) => {
        await compiler.setFileContentsAsync(
          'lib/_third_party_main.js',
          compiler.code()
        )
        compiler.options.empty = true;
        return next()
      }
    ]
  }
```

**Anything else we need to know?:**

**Environment**

- Platform(OS/Version): Windows 10
- Host Node Version: 8.11.4
- Target Node Version: 8.11.4
- Nexe version: 3.0.0-beta.13
- Python Version: 2.7

**Comment by staifan** [comment]
Hello,

I'm having the same problem as **andytuch** when I launch my app

> _third_party_main.js:25
    var _a = binary.layout, resourceStart = _a.resourceStart, stat = _a.stat;

> TypeError:
 Cannot read property 'resourceStart' of undefined
    at shimFs (_third_party_main.js:25:48)
    at _third_party_main.js:309:1
    at _third_party_main.js:309:25
    at NativeModule.compile (internal/bootstrap/loaders.js:364:7)
    at Function.NativeModule.require (internal/bootstrap/loaders.js:176:18)
    at process.nextTick (internal/bootstrap/node.js:249:22)
    at process._tickCallback (internal/process/next_tick.js:61:11)

Environment :

Platform(OS/Version): Windows 10
Host Node Version: 10.15.3
Target Node Version: 10.15.3
Nexe version: 3.0.0-beta.13
Python Version: 2.7

Do you have a track about this problem?

**Comment by staifan** [comment]
Sorry, my version of Nexe is 3.0.0-beta.15

**Comment by calebboyd** [comment]
@staifan are you signing it as well? 

I suspect this is the issue with @andytuch as well

**Comment by staifan** [comment]
yes, that's why I use the patch _third_party_main.js, I thought it was what had to be done to be able to sign the file of my app because otherwise I get the problem of the subject:

_third_party_main.js: 12 throw 'Invalid Nexe binary' ^ Invalid Nexe binary # 372

**Comment by calebboyd** [comment]
This should be fixed after since #600
